recipe-server/client/control/components/action_fields/HeartbeatFields.js

```diff
@@ -117,6 +121,6 @@ export class HeartbeatFields extends ActionFields {
 
 export default connect(
   state => ({
-    recipeArguments: selector(state, 'arguments'),
+    recipeArguments: selector(state, 'arguments') || {},
```

Comment:
I think this should go to master, since it isn't related to preference rollouts.